frontend/src/lib/hooks/use-project-list.tsx

```diff
@@ -24,3 +24,20 @@ export function useProjectList(filters: ProjectSearch) {
     }
   });
 }
+
+export function useArchivedProject() {
+  return useQuery({
+    queryKey: projectKeys.archived,
+    queryFn: ({ signal }) => {
+      return apiClient.GET("/api/archived-projects/", {
+        signal
+      });
+    },
+    refetchInterval: (query) => {
+      if (query.state.data?.data?.results) {
+        return TEN_SECONDS;
+      }
+      return false;
+    }
```

Comment:
You can remove the refetch interval, we don't need to refetch archived projects on a regular basis.